### DISCUSSION TITLE
Jak usunąć stacje z wykazu?

### DISCUSSION THREAD

**Discussion by ghost** [TomaszGasior/RadioLista-v3#180] [3. Inne Other]
Zdałem sobie sprawe, ze z wykazu musze usunąć cos. Nie mogłem znaleźć takiej opcji. Czy jest możliwość usunięcia stacji z wykazu?

**Comment by TomaszGasior** [comment]
Witaj! :) Musisz wejść w ustawienia wykazu i tam w menu u góry znajdziesz usuwanie wykazu lub stacji.

**Reply by TomaszGasior** [reply]
PS Proszę następnym razem o wybranie właściwej kategorii. Dzieki. :)

**Reply by TomaszGasior** [reply]
Funkcja usuwania stacji jest umieszczona w nieintuicyjnym i trudnym do znalezienia miejscu. Mam tego świadomość i w nowym designie (#116) będzie to łatwiejsze.

**Comment by TomaszGasior** [comment]
@era2750 Rzuć okiem na nowy design serwisu. Mam nadzieję, że w nowej formie usuwanie stacji w wykazie przypadnie ci do gustu. :) https://github.com/TomaszGasior/RadioLista-v3/discussions/139#discussioncomment-2893918
